pkgs/by-name/id/idb-companion/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "idb-companion";
+  version = "1.1.8";
+
+  src = fetchurl {
+    url = "https://github.com/facebook/idb/releases/download/v${version}/idb-companion.universal.tar.gz";
+    hash = "sha256-O3LMappbGiKhiCBahAkNOilDR6hGGA79dVzxo8hI4+c=";
+  };
+
+  unpackPhase = ''
+    tar -xzf $src
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r ./* $out/bin/
+  '';
+
+  meta = with lib; {
+    description = "A powerful command line tool for automating iOS simulators and devices";
+    homepage = "https://github.com/facebook/idb";
+    license = licenses.mit;
+    platforms = platforms.darwin;
+    mainProgram = "idb-companion";
```

Comment:
```sh
$ nix run -L .#idb-companion
error: unable to execute '/nix/store/z7d4gl49849arqigd4dkrksa6vq3a5p0-idb-companion-1.1.8/bin/idb-companion': No such file or directory
```

Reply:
we probably do not need `mainProgram = "idb-companion";`

Reply:
hmm @sikmir : can you please elaborate more on what this means?

Reply:
There is no such binary `idb-companion` in the bin. I see only `idb-companion.universal` in the bin. I'm not sure what is main program here, maybe `bin/idb-companion.universal/bin/idb_companion`. But in such case it should be symlinked into bin.

Reply:
Thanks for pointing that out. 
Indeed the `idb_companion` I needed was inside the path you mentioned.
I added a symlink step to install phase.